### ISSUE TITLE
Subject: [shadow-support] Question regarding usage of shadow - running


### ISSUE THREAD

**Issue by shadow-org-bot** [shadow/shadow#611]
From: Andrew Asfaganov <andrew.asfaganov -at- gmail.com>
To: shadow-support -at- cs.umn.edu
Subject: [shadow-support] Question regarding usage of shadow - running
Date: Sun, 15 May 2016 20:10:24 +1000

Hi guys,
Quick question. I've been reading the wiki on Github for Shadow and it
explains quite a lot, I've installed it, run some example experiment and
had a look at the extra plugins, however, if I'm trying to run a python
script on each node, do I still need to create a plugin for it? If so, do
you have any example plugins for programs written in python?

What I like about Shadow, at least from my understanding, is that it
doesn't create a virtual instance for each node, rather runs the code on
the host itself, which is more lightweight, especially for my purpose,
where all I'm trying to do is run a single process on each node.

Also, just want to confirm, I can't run Shadow across multiple hardware
machines and for them to share the load of a single experiment, is that
correct?

Basically, for my project, I need to emulate a network to run about a
billion nodes executing a python script, communicating p2p. Any help is
much appreciated!

Thanks!
-- 
Andrew A.
+61447489587
